test/src/main/java/org/apache/accumulo/test/zookeeper/fate/ZooMutatorIT.java

```diff
@@ -75,12 +73,10 @@
    */
   @Test
   public void concurrentMutatorTest() throws Exception {
-    try (var client = Accumulo.newClient().from(getClientProps()).build();
-        var context = (ClientContext) client) {
-      String secret = cluster.getSiteConfiguration().get(Property.INSTANCE_SECRET);
 
-      ZooReaderWriter zk = new ZooReaderWriter(context.getZooKeepers(),
-          context.getZooKeepersSessionTimeOut(), secret);
+    try (ZooKeeperTestingServer szk = new ZooKeeperTestingServer();) {
```

Comment:
nit: extra semicolon

```suggestion
    try (ZooKeeperTestingServer szk = new ZooKeeperTestingServer()) {
```